src/python/tests/test_filesys.py

```diff
@@ -65,10 +66,10 @@ class TestDirCmp(BaseTestFilesys):
     """Tests :class:`DirCmp` class."""
 
     @pytest.mark.dependency(name='test_init', scope='class')
-    def test_init(self, tmp_dir: Path) -> None:
+    def test_init(self, tmp_path: Path) -> None:
```

Comment:
This fixture is no longer used in this method:
```suggestion
    def test_init(self) -> None:
```